larpandoracontent/LArMonitoring/VisualMonitoringAlgorithm.cc

```diff
@@ -150,6 +151,23 @@ void VisualMonitoringAlgorithm::VisualizeMCParticleList(const std::string &listN
         }
     }
 
+    for (const MCParticle *pMCParticle : *pMCParticleList)
+    {
+        const LArMCParticle *const pLArMCParticle(dynamic_cast<const LArMCParticle*>(pMCParticle));
+        const std::vector<pandora::CartesianVector> steps(pLArMCParticle->GetMCStepPositions());
+
+        for (unsigned int step = 0; step < steps.size() - 1; ++step)
```

Comment:
What happens here if `steps` is empty? Do we try to take 1 away from 0 and assign it to an unsigned int?